### PR TITLE
Feature/iot arduino code

### PR DESCRIPTION
**Pull request by Hunter27**
created multiple sensor functionality and basic fault tolerance procedures

### CODE REVIEW

File: Water sensors code/Arduino code/MonitorReading/MonitorReading.ino [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/*
+Liquid flow rate sensor -DIYhacking.com Arvind Sanjeev
+
+Measure the liquid/water flow rate using this code. 
+Connect Vcc and Gnd of sensor to arduino, and the 
+signal line to arduino digital pin 2.
+ 
+ */
+
+byte statusLed    = 13;
```
Comment (CilliersN) [inline Water sensors code/Arduino code/MonitorReading/MonitorReading.ino:10]: these spaces tho

File: WaterLog_Backend/Properties/launchSettings.json.orig [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (CilliersN) [inline WaterLog_Backend/Properties/launchSettings.json.orig]: remove this file

**Review by ash-rabbit — APPROVED** [review]

File: Water sensors code/IoT Device/example.c [hunk 3]
```diff
@@ -175,6 +198,21 @@ int main(void)
     cr = Client_register(client, topicName1, &topic);
     ASSERT(cr == CLIENT_SUCCESS);
     DEBUGOUT("publishing\n");
+    int count =0;
+    faulty = false;
+    for (int i=0;i<2;i++){
+        GPIO_WritePinOutput(GPIOA, 19, i);
+        getMessageSend(count);
+        cr = Client_publish(client, topic, MQTT_QOS1, false,
```
Comment (CilliersN) [inline Water sensors code/IoT Device/example.c]: multiline. 
apply indentation

Comment (CilliersN) [inline Water sensors code/IoT Device/example.c:207]: multiline. 
apply indentation

**Review by CilliersN — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Properties/launchSettings.json.orig (hunk 2 above)
Comment (Hunter27) [reply WaterLog_Backend/Properties/launchSettings.json.orig]: done

File: Water sensors code/Arduino code/MonitorReading/MonitorReading.ino (hunk 1 above)
Comment (Hunter27) [reply Water sensors code/Arduino code/MonitorReading/MonitorReading.ino:10]: fixed all

File: Water sensors code/IoT Device/example.c (hunk 3 above)
Comment (Hunter27) [reply Water sensors code/IoT Device/example.c:207]: fixed this

**Pull request merged**